aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -77,96 +77,92 @@ dotnet new mvc
-
-When **Folder** is selected, specify a folder path to store the published assets. The default folder is *bin\Release\PublishOutput*. Click the **Create Profile** button to finish.
-
-Once a publish profile is created, the **Publish** tab changes. The newly created profile appears in a drop-down list. Click **Create new profile** to create another new profile.
-
-![The Publish tab of the app capacities page showing FolderProfile](visual-studio-publish-profiles/_static/create_new.png)
-
-The Publish wizard supports the following publish targets:
+The **Publish** tab of the app capacities page is displayed. If the project lacks a publish profile, the **Pick a publish target** page is displayed. You're asked to select one of the following publish targets:
 
 * Azure App Service
+* Azure App Service on Linux
 * Azure Virtual Machines
-* IIS, FTP, etc. (for any web server)
 * Folder
+* IIS, FTP, Web Deploy (for any web server)
 * Import Profile
 
-For more information, see [What publishing options are right for me](/visualstudio/ide/not-in-toc/web-publish-options).
+To determine the most appropriate publish target, see [What publishing options are right for me](/visualstudio/ide/not-in-toc/web-publish-options).
+
+When the **Folder** publish target is selected, specify a folder path to store the published assets. The default folder path is *bin\\{PROJECT CONFIGURATION}\\{TARGET FRAMEWORK MONIKER}\publish\\*. For example, *bin\Release\netcoreapp2.2\publish\\*. Click the **Create Profile** button to finish.
+
+Once a publish profile is created, the **Publish** tab's content changes. The newly created profile appears in a drop-down list. Below the drop-down list, select **Create new profile** to create another new profile.
 
-When creating a publish profile with Visual Studio, a *Properties/PublishProfiles/{PROFILE NAME}.pubxml* MSBuild file is created. The *.pubxml* file is a MSBuild file and contains publish configuration settings. This file can be changed to customize the build and publish process. This file is read by the publishing process. `<LastUsedBuildConfiguration>` is special because it's a global property and shouldn't be in any file that's imported in the build. See [MSBuild: how to set the configuration property](http://sedodream.com/2012/10/27/MSBuildHowToSetTheConfigurationProperty.aspx) for more information.
+Visual Studio's publish tool produces a *Properties/PublishProfiles/{PROFILE NAME}.pubxml* MSBuild file describing the publish profile. The *.pubxml* file:
+
+* Contains publish configuration settings and is consumed by the publishing process.
+* Can be modified to customize the build and publish process.
+
+`<LastUsedBuildConfiguration>` is special because it's a global property and shouldn't be in any file that's imported in the build. For more information, see [MSBuild: how to set the configuration property](http://sedodream.com/2012/10/27/MSBuildHowToSetTheConfigurationProperty.aspx).
```

Comment:
```suggestion
`<LastUsedBuildConfiguration>` is special property because it's a global property and shouldn't be in any file that's imported during the build. For more information, see [MSBuild: how to set the configuration property](http://sedodream.com/2012/10/27/MSBuildHowToSetTheConfigurationProperty.aspx).
```